### PR TITLE
Feature/add database

### PR DESCRIPTION
**Pull request by dangoth**
Added a docker image with the database + updated EmployeeController to use it

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
```
Comment (DmytroDorodnykhTeamInternational) [inline .vscode/launch.json]: Do we need vscode in the project? I think that we need to add it to gitignore

Comment (dangoth) [reply .vscode/launch.json]: removed and added to .gitignore

**Review by Ygoraphael — APPROVED** [review]
well Done

File: Portal_Perfomance_Employes/Controllers/EmployeeController.cs [hunk 2]
```diff
@@ -9,53 +10,57 @@ namespace PortalPerfomanceEmployees.Controllers
...
+            var emp = await _context.Employees
+                .FirstOrDefaultAsync(e => e.Id == id);
+            return emp == null ? NotFound("Employee with that ID was not found") : Ok(emp) ;
         }
 
         [HttpPost]
-        public async Task<IEnumerable<Employee>> CreateEmployee(Employee employee)
+        public async Task<IActionResult> CreateEmployee(Employee employee)
         {
-            _employeesDB.employees.Add(employee);
```
Comment (VladyslavCherednychenko) [inline Portal_Perfomance_Employes/Controllers/EmployeeController.cs:34]: I encountered a problem that if you change the id from zero to any number in the POST request, it causes an Error: response status is 500

I solved the problem by adding `employee.Id = 0;` before `_context.Employees.Add(employee);`
image


And the result before add that code: 
image

And After: 
image

**Review by VladyslavCherednychenko — CHANGES_REQUESTED** [review]

File: Portal_Perfomance_Employes/Controllers/EmployeeController.cs (hunk 2 above)
Comment (mbogatkiewicz) [reply Portal_Perfomance_Employes/Controllers/EmployeeController.cs:34]: (hopefully) resolved by introducing an `EmployeeDTO` as the POST method body. The `Id` and `Created` fields should no longer be required and should automatically be assigned on the backend

Comment (dangoth) [reply Portal_Perfomance_Employes/Controllers/EmployeeController.cs:34]: (hopefully) resolved by introducing an EmployeeDTO as the POST method body. The Id and Created fields should no longer be required and should automatically be assigned on the backend

**Review by DmytroDorodnykhTeamInternational — APPROVED** [review]

**Pull request merged**